{
  "uid" : "57138e756ee0dc7",
  "name" : "test_mul_False[get_mul_datas_False0]",
  "fullName" : "test_calc2_homework.TestCalc2#test_mul_False",
  "historyId" : "efb3c2c7c547b9b17a2e2c971bd2a74c",
  "time" : {
    "start" : 1614853637526,
    "stop" : 1614853637527,
    "duration" : 1
  },
  "status" : "broken",
  "statusMessage" : "NameError: name 'aaaaaa' is not defined",
  "statusTrace" : "self = <testing.test_calc2_homework.TestCalc2 object at 0x000000000445EA58>\nget_calc = <python_code.calc.Calculator object at 0x0000000003BD7780>, get_mul_datas_False = [0, 0, 1]\n\n    @allure.story(\"测试乘法_无效测试用例\")\n    @pytest.mark.mul\n    @pytest.mark.run(order=3)\n    def test_mul_False(self, get_calc, get_mul_datas_False):\n        with allure.step(\"计算两个数 相乘积结果\"):\n            result=get_calc.mull(get_mul_datas_False[0],get_mul_datas_False[1])\n        with allure.step(\"判断 相乘积的数 是否为 float类型 \"):\n            if isinstance(result,float):\n                with allure.step(\"相乘积的数 为float类型，result 保留小数点后两位\"):\n                    result=round(result,2)\n        with allure.step(\"断言 计算结果 等于 预期结果\"):\n            assert result!=get_mul_datas_False[2]\n>           print(aaaaaa)\nE           NameError: name 'aaaaaa' is not defined\n\ntest_calc2_homework.py:258: NameError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "get_calc",
    "time" : {
      "start" : 1614853636617,
      "stop" : 1614853636618,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false
  }, {
    "name" : "get_mul_datas_False",
    "time" : {
      "start" : 1614853637524,
      "stop" : 1614853637524,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "NameError: name 'aaaaaa' is not defined",
    "statusTrace" : "self = <testing.test_calc2_homework.TestCalc2 object at 0x000000000445EA58>\nget_calc = <python_code.calc.Calculator object at 0x0000000003BD7780>, get_mul_datas_False = [0, 0, 1]\n\n    @allure.story(\"测试乘法_无效测试用例\")\n    @pytest.mark.mul\n    @pytest.mark.run(order=3)\n    def test_mul_False(self, get_calc, get_mul_datas_False):\n        with allure.step(\"计算两个数 相乘积结果\"):\n            result=get_calc.mull(get_mul_datas_False[0],get_mul_datas_False[1])\n        with allure.step(\"判断 相乘积的数 是否为 float类型 \"):\n            if isinstance(result,float):\n                with allure.step(\"相乘积的数 为float类型，result 保留小数点后两位\"):\n                    result=round(result,2)\n        with allure.step(\"断言 计算结果 等于 预期结果\"):\n            assert result!=get_mul_datas_False[2]\n>           print(aaaaaa)\nE           NameError: name 'aaaaaa' is not defined\n\ntest_calc2_homework.py:258: NameError",
    "steps" : [ {
      "name" : "计算两个数 相乘积结果",
      "time" : {
        "start" : 1614853637526,
        "stop" : 1614853637526,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "hasContent" : false,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false
    }, {
      "name" : "判断 相乘积的数 是否为 float类型 ",
      "time" : {
        "start" : 1614853637526,
        "stop" : 1614853637526,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "hasContent" : false,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false
    }, {
      "name" : "断言 计算结果 等于 预期结果",
      "time" : {
        "start" : 1614853637526,
        "stop" : 1614853637526,
        "duration" : 0
      },
      "status" : "broken",
      "statusMessage" : "NameError: name 'aaaaaa' is not defined\n",
      "statusTrace" : "  File \"F:\\study\\pytest_shizhan2\\testing\\test_calc2_homework.py\", line 258, in test_mul_False\n    print(aaaaaa)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "hasContent" : true,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true
    } ],
    "attachments" : [ {
      "uid" : "7f937b9bf029561d",
      "name" : "stdout",
      "source" : "7f937b9bf029561d.txt",
      "type" : "text/plain",
      "size" : 92
    } ],
    "parameters" : [ ],
    "stepsCount" : 3,
    "hasContent" : true,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true
  },
  "afterStages" : [ {
    "name" : "get_calc::0",
    "time" : {
      "start" : 1614853638021,
      "stop" : 1614853638021,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false
  }, {
    "name" : "get_mul_datas_False::0",
    "time" : {
      "start" : 1614853637537,
      "stop" : 1614853637537,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false
  } ],
  "labels" : [ {
    "name" : "feature",
    "value" : "测试计算器:(加减乘除)"
  }, {
    "name" : "story",
    "value" : "测试乘法_无效测试用例"
  }, {
    "name" : "tag",
    "value" : "mul"
  }, {
    "name" : "tag",
    "value" : "run(order=3)"
  }, {
    "name" : "suite",
    "value" : "test_calc2_homework"
  }, {
    "name" : "subSuite",
    "value" : "TestCalc2"
  }, {
    "name" : "host",
    "value" : "KD4JP1FBJZM3YKK"
  }, {
    "name" : "thread",
    "value" : "6444-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_calc2_homework"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "get_mul_datas_False",
    "value" : "[0, 0, 1]"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "run(order=3)", "mul" ]
  },
  "source" : "57138e756ee0dc7.json",
  "parameterValues" : [ "[0, 0, 1]" ]
}